book/10-git-internals/sections/objects.asc

```diff
@@ -126,7 +126,7 @@ The next type of Git object we'll examine is the _tree_, which solves the proble
 Git stores content in a manner similar to a UNIX filesystem, but a bit simplified.
 All the content is stored as tree and blob objects, with trees corresponding to UNIX directory entries and blobs corresponding more or less to inodes or file contents.
 A single tree object contains one or more entries, each of which is the SHA-1 hash of a blob or subtree with its associated mode, type, and filename.
-For example, the most recent tree in a project may look something like this:
+For example, a repository (different from the one created in the previous section) may have a tree that looks something like:
```

Comment:
I like where you're going here. What if we did this instead?

```suggestion
For example, let's say you have a project where the most-recent tree looks something like:
```

Reply:
Your suggestion reads much better!